services/core/PlatformDriverAgent/platform_driver/interfaces/home_assistant.py

```diff
@@ -57,7 +61,7 @@ def __init__(self, read_only, pointName, units, reg_type, attributes, entity_id,
 def _post_method(url, headers, data, operation_description):
     err = None
     try:
-        response = requests.post(url, headers=headers, json=data)
+        response = requests.post(url, headers=headers, json=data, verify=self.verify_option)
```

Comment:
What is verify_option ...I don't see this anywhere in the code.

Reply:
its on line 107. This is for the requests library, verify_option will eaither be a path to a crt file or will be False and will not use SSL.